dvc/remote/local/__init__.py

```diff
@@ -564,5 +589,97 @@ def unprotect(path_info):
             RemoteLOCAL._unprotect_file(path)
 
     @staticmethod
-    def protect(path_info):
-        os.chmod(path_info.path, stat.S_IREAD | stat.S_IRGRP | stat.S_IROTH)
+    def protect(path):
```

Comment:
But RemoteBASE.protect still has old argument type. Let's keep it as it was before, since it is used outside of RemoteLOCAL.